app/templates/power-ups/topics.html

```diff
@@ -183,6 +183,7 @@ <h2 id="data-storage">Data Storage</h2>
   <pre>t.set('board', 'private', 'key', value)</pre>
   <p>or</p>
   <pre>t.set('board', 'private', { key1: value1, key2: value2 })</pre>
+  <p>Storing at the 'organization' scope will only work if the active user is a member of the team to which the board belongs.</p>
```

Comment:
Maybe word it as:

`Storing at the 'organization' scope will only work if the both the board is in a team and the active user is a member of the team to which the board belongs.`